src/sentry/api/endpoints/organization_environment_details.py

```diff
@@ -0,0 +1,53 @@
+from django.db import IntegrityError
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.models import Environment, EnvironmentBookmark
+
+
+class OrganizationEnvironmentSerializer(serializers.Serializer):
+    isBookmarked = serializers.BooleanField()
+
+
+class OrganizationEnvironmentDetailsEndpoint(OrganizationEndpoint):
+    def get(self, request: Request, organization, environment) -> Response:
```

Comment:
Could you document this endpoint? https://develop.sentry.dev/api/public/#how-to-make-an-endpoint-public